BASE/src/Microsoft.ApplicationInsights/Extensibility/Implementation/RichPayloadEventSource.cs

```diff
@@ -21,7 +21,11 @@ internal sealed partial class RichPayloadEventSource : IDisposable
         internal readonly EventSource EventSourceInternal;
 
         /// <summary>Event provider name.</summary>
+#if REDFIELD
+        private const string EventProviderName = "Redfield-Microsoft-ApplicationInsights-Data";
```

Comment:
Partners like Azure profiler team needs to be aware of this?

Reply:
Todd mentioned some azure app service diagnostics also depend on the provider name.

Reply:
IIRC, profiler team has already updated their side last year to listen to this eventsource. But we can double check with Paul.

Reply:
Good call out. Documents needs to be updated to collect ETW traces from both SDK and Redfield. We cannot expect customer to switch to appropriate set of events for Nuget and auto-instrumentation approach. Will track this in a separate internal item and get docs updated when auto-instrumentation goes live with new package.